tests/func/experiments/test_show.py

```diff
@@ -48,18 +48,6 @@ def make_executor_info(**kwargs):
     return ExecutorInfo(**kwargs)
 
 
-@pytest.mark.vscode
```

Comment:
Will this be an issue for the vscode extension?

Reply:
VS Code already doesn't show the table in this scenario, but good callout that we need to check this doesn't break anything with @iterative/vs-code.

Reply:
@mattseddon Will it be a problem if `exp` commands return an error in an empty repo (one with no commits)?